rust/arrow/src/buffer.rs

```diff
@@ -676,9 +676,9 @@ impl MutableBuffer {
     /// `new_len` will be zeroed out.
     ///
     /// If `new_len` is less than `len`, the buffer will be truncated.
-    pub fn resize(&mut self, new_len: usize) -> Result<()> {
+    pub fn resize(&mut self, new_len: usize) -> () {
```

Comment:
```suggestion
    pub fn resize(&mut self, new_len: usize) {
```

It is the same in other positions.